docs/src/main/asciidoc/security-oidc-bearer-token-authentication.adoc

```diff
@@ -17,43 +17,44 @@ Secure HTTP access to Jakarta REST (formerly known as JAX-RS) endpoints in your
 
 Quarkus supports the Bearer token authentication mechanism through the Quarkus OpenID Connect (OIDC) extension.
 
-The bearer tokens are issued by OIDC and OAuth 2.0 compliant authorization servers, such as link:https://www.keycloak.org[Keycloak].
+The bearer tokens are issued by OIDC and OAuth 2.0-compliant authorization servers, such as link:https://www.keycloak.org[Keycloak].
```

Comment:
Hi @sheilamjones, as you confirmed with `currently authenticated` vs `currently-authenticated`, the expression without `-`  should work fine, so I propose to apply a form without `-`, I probably  would not have noticed if it were done as `-` originally, but it does look like the `-` is not used a lot

Reply:
Thanks @sberyozkin. Both ways are grammatically correct. I shall remove the hyphen again in this instance.